server/server_test.go

```diff
@@ -430,17 +424,17 @@ func TestDisconnectingServer(t *testing.T) {
 		return
 	}
 
-	time.Sleep(500 * time.Millisecond)
+	time.Sleep(idleTimeout)
```

Comment:
The value of this hasn't changed, I'm just referring to the variable itself to clarify that we're sleeping the idleTimeout.